docs/environment-variables.md

```diff
@@ -31,6 +31,7 @@ This document outlines environment variables that can be used to customize behav
 | `DISABLE_MAX_RECURSION`                  | `bool`              | `false`                                                                                     | Set to true to disable the recursion preventer, which will stop a workflow running which has called into a child template 100 times                                                                                                                                      |
 | `EXPRESSION_TEMPLATES`                   | `bool`              | `true`                                                                                      | Escape hatch to disable expression templates.                                                                                                                                                                                                                            |
 | `EVENT_AGGREGATION_WITH_ANNOTATIONS`     | `bool`              | `false`                                                                                     | Whether event annotations will be used when aggregating events.                                                                                                                                                                                                          |
+| `EXECUTOR_SIGNAL_TIMEOUT`                | `time.Duration`     | `10m`                                                                                       | When the executor does not finish after this time, it will times out and simply returns an err. I'm happy with 10 minutes.                                                                                                                                               |
```

Comment:
looks this was a copy+paste -- that would not be an appropriate documentation line
```suggestion
| `EXECUTOR_SIGNAL_TIMEOUT`                | `time.Duration`     | `10m`                                                                                       | Timeout for a signal to the executor.                                                                                                                                               |
```

What Alan suggested [above](https://github.com/argoproj/argo-workflows/pull/13012#discussion_r1593644855) was:
```suggestion
| `EXECUTOR_SIGNAL_TIMEOUT`                | `time.Duration`     | `10m`                                                                                       | How long to wait after the executor has been terminated before timing out.                                                                                                                                               |
```

I made a small modification to that as well

Reply:
Thanks for helping me revise it.

Reply:
Thanks for fixing this.